src/hub/install-task.ts

```diff
@@ -81,13 +84,18 @@ async function doInstallTask(task: HubTaskInstallation): Promise<boolean> {
       }
 
       if (result.error){
-        vscode.window.showWarningMessage(`Task installation failed: ${getStderrString(result.error)}`);
+        message = `Task installation failed: ${getStderrString(result.error)}`;
+        telemetryLogError('tekton.hub', message);
```

Comment:
Are you sure that `getStderrString(result.error)` doesn't contains user sensitive information?
Like IP, local paths etc.

Reply:
I have hide the local path. It was coming in error when we install the clustertask from tektonHub.